schema/author.js

```diff
@@ -18,6 +18,7 @@ const AuthorType = new GraphQLObjectType({
     href: {
       type: GraphQLString,
       resolve: ({ profile_handle }) => `/${profile_handle}`,
+      deprecationReason: "Profiles and thus artist hrefs don't exist anymore",
```

Comment:
@1aurabrown Assuming this is the case, I’ve added this commit to my PR https://github.com/artsy/metaphysics/pull/313/commits/a62db2d449935bc2007b342d92acae17ec431e7d